pages/core-concepts/seed.mdx

```diff
@@ -145,19 +114,81 @@ await snaplet.posts([{
 Here is what its store looks like internally:
 
 ```ts
-const store = {
+const postsStore = {
   users: [user, user, user, user],
   posts: [post],
   comments: [comment, comment, comment],
 }
 ```
 
-This store is very useful when you want to [connect](/core-concepts/seed#connecting-to-existing-data) your models, reusing existing data instead of generating new one.
+#### The global store
+
+The global store is a special store which contains all the data that was generated as part of the plans you executed.
+It is accessible under the `$store` property of the data client.
+
+```ts
+await snaplet.posts(x => x(3));
+
+// { posts: [post, post, post], users: [user, user, user] }
+console.log(snaplet.$store);
+```
+
+#### Local stores
+
+Each plan produces its own store and returns it when executed.
+It's particularly useful for the [`connect` option](/core-concepts/seed#using-connect-option).
+
+```ts
+const postsStore = await snaplet.posts(x => x(3));
+
+// { posts: [post, post, post], users: [user, user, user] }
+console.log(postsStore);
+
+const usersStore = await snaplet.users(x => x(2));
+
+// { users: [user, user] }
+console.log(usersStore);
+
+// { posts: [post, post, post], users: [user, user, user, user, user] }
+console.log(snaplet.$store);
+```
+
+#### Client state
 
+The `SnapletClient` keeps track of what was created before with `snaplet.$store` and increments global seeds each time you call a model function.
```

Comment:
> increments global seeds each time you call a model function.
Not for this PR, but I wonder if we need a section in the docs where we explain what does and does not change the global seeds, so people understand how the determinism works.

Reply:
Hmmm it could be useful yes!